src/components/freelancerJobs/freelancerPage.tsx

```diff
@@ -113,19 +131,17 @@ const FreelancerPage: FC = () => {
 									}
 								})
 								.filter((post: IPost) => {
-									const skill = post.jobSkills.map(skill => {
+									const jobSkills = post.jobSkills.map(skill => {
 										return skill.name;
 									});
 									if (
-										post.jobCategory.name === userChoice.label &&
-										slider[0] <= post.fromHourRate &&
-										slider[1] >= post.fromHourRate &&
-										JSON.stringify(radio) === JSON.stringify(post.jobDuration) &&
-										JSON.stringify(skill).includes(JSON.stringify(skills))
+										userPrice[0] <= post.fromHourRate &&
```

Comment:
why price in array?

Reply:
because I need min and max slider values so they contain in array